src/Database/Dialect/SqliteDialectTrait.php

```diff
@@ -120,7 +120,7 @@ protected function _transformFunctionExpression(FunctionExpression $expression)
                     ->setConjunction(' ,')
                     ->iterateParts(function ($p, $key) {
                         if ($key === 0) {
-                            $value = rtrim(strtolower($p), 's');
+                            $value = strtolower(rtrim($p, 's'));
```

Comment:
This is not functionally same. Earlier a capital `S` at end of string would have been trimmed, now it won't be.

Reply:
Hm, good catch. A false positive, reverting.